men_team_1/db/migrate/20181109003112_create_pairings.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePairings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :pairings do |t|
+      t.string :status
+      t.references :mentor_id, :mentor, index: true, foreign_key: true
+      t.references :mentee_id, :mentee, index: true, foreign_key: true
```

Comment:
FYI, `foreign_key: true` true creates an index implicitly, so `index: true` is redundant. It doesn't hurt anything to have both, though.

Reply:
Also, I'm pretty sure your `references` calls have too many parameters. You just need `:mentor` not `:mentor_id, :mentor`. `references` creates a new column with the reference's name followed by `_id`.

Reply:
@dbechrd how does it know to reference the user id column in the Users table? Is the way in file men_team_1/db/migrate/20181109011841_add_references_to_pairing.rb below how it should be set up?

Reply:
This is a little rails magic here. @dbechrd is right. `t.references` can be sludgy. 

```ruby
# migration file
...
t.integer :shopper_id
...

# model
belongs_to :mentor, class_name: 'User'
```

Rails is smart enough to pair up the `mentor_id` column with your association to `user` aliased as `mentor` as defined by the association.